src/main/java/be/cytomine/client/models/Software.java

```diff
@@ -29,4 +31,21 @@ public Software(String name, String serviceName, String resultType, String execu
         this.set("resultName", resultType);
         this.set("executeCommand", executeCommand);
     }
+    public Software deprecateSoftware(Long id) throws CytomineException {
```

Comment:
It seems more meaningful to call the method `deprecate` 
```
Software mySoft = new Software(...)
mySoft.save()
mySoft.deprecate()
```

Reply:
done